engine/src/main/java/org/destinationsol/menu/LoadingScreen.java

```diff
@@ -49,7 +50,7 @@ public void updateCustom(SolApplication solApplication, SolInputManager.InputPoi
 
     @Override
     public void drawText(UiDrawer uiDrawer, SolApplication solApplication) {
-        uiDrawer.drawString("Loading...", displayDimensions.getRatio() / 2, .5f, FontSize.MENU, true, SolColor.WHITE);
+        uiDrawer.drawString(Translation.translate("${core:loading#loading}") + "...", displayDimensions.getRatio() / 2, .5f, FontSize.MENU, true, SolColor.WHITE);
```

Comment:
I wonder if the `...` here should be made part of the translation string and not hard coded. The way to indicate a pending state may vary between languages. For instance imagine a language like Japanese.